app/views/editorial/sections/show.html.haml

```diff
@@ -24,7 +24,7 @@
     - @section.nodes.order(updated_at: :desc).each do |page|
       %tr
         %td
-          =link_to "#{page.name}", editorial_section_node_path(@section, page.id)
+          =link_to (page.name.blank? ? "[no name]" : "#{page.name}") , editorial_section_node_path(@section, page.id)
```

Comment:
Could you use a decorator method to replace the ternary operator here? Its very hard to test this logic otherwise